coretex/cli/modules/node.py

```diff
@@ -62,5 +79,61 @@ def shouldUpdate(repository: str, tag: str) -> bool:
             if repository in digest and manifestJson["Descriptor"]["digest"] in digest:
                 return False
         return True
-    except:
-        return False
+    except CommandException:
+        return True
+
+
+def registerNode(name: str) -> str:
+    params = {
```

Comment:
No need to define `params` separately, pass the dictionary directly to the function